gameModel/pages/42.json

```
@@ -533,6 +533,78 @@
+                        "onVariableChange" : {
+                          "client" : {
+                            "@class" : "Script",
+                            "content" : "import { updateRequestValue } from \"./UIfacade/methaneFacade\";\n\nupdateRequestValue(Context.item.type, Context.value);",
+                            "language" : "TypeScript"
+                          }
+                        },
+                        "script" : {
+                          "@class" : "Script",
+                          "content" : "import { getRequestValue } from \"./UIfacade/methaneFacade\";\n  \tgetRequestValue<boolean>(Context.item.type);",
+                          "language" : "TypeScript"
+                        },
+                        "style" : { },
+                        "type" : "checkbox"
+                      },
+                      "type" : "Boolean"
+                    }, {
+                      "props" : {
+                        "className" : "methane-input-label",
+                        "layoutStyle" : {
+                          "paddingBottom" : "0px",
+                          "paddingTop" : "0px",
+                          "width" : "84px"
+                        },
+                        "onCancel" : { },
+                        "onVariableChange" : { },
+                        "style" : { },
+                        "text" : {
+                          "@class" : "Script",
+                          "content" : "// TODO transalations\nContext.item.type;",
+                          "language" : "TypeScript"
+                        }
+                      },
+                      "type" : "Text"
+                    } ]
+                  },
+                  "type" : "FlexList"
+                } ],
+                "exposeAs" : "item",
+                "getItemsFn" : {
+                  "@class" : "Script",
+                  "content" : "import { getFilteredResourceContainerTypeArray } from \"./UIfacade/methaneFacade\";\ngetFilteredResourceContainerTypeArray(true);",
+                  "language" : "TypeScript"
+                },
+                "itemKey" : "type",
+                "layout" : { },
+                "layoutStyle" : {
+                  "marginTop" : "7px"
+                },
+                "style" : {
+                  "flexWrap" : "wrap",
+                  "maxWidth" : "700px"
+                }
+              },
+              "type" : "For each"
+            }, {
+              "props" : {
+                "children" : [ {
+                  "props" : {
+                    "children" : [ {
+                      "props" : {
+                        "disableIf" : {
+                          "@class" : "Script",
+                          "content" : "import { hasContainerOfType } from \"./game/common/resources/emergencyDepartment\";\nimport { containerTypeAvailable } from \"./UIfacade/methaneFacade\";\n!containerTypeAvailable(Context.item.type)",
```

```suggestion
                          "content" : "import { containerTypeAvailable } from \"./UIfacade/methaneFacade\";\n!containerTypeAvailable(Context.item.type)",
```

first import is not used